sklearn/ensemble/weight_boosting.py

```diff
@@ -657,7 +656,7 @@ def decision_function(self, X):
             class in ``classes_``, respectively.
         """
         check_is_fitted(self, "n_classes_")
-        X = self._validate_X_predict(X)
+        self._validate_data(X)
```

Comment:
I am surprised you don't need to use the output like before. Did I miss something?

Reply:
You are right.